### DISCUSSION TITLE
I am a verified student having trouble redeeming my Global Campus offers

### DISCUSSION THREAD

**Discussion by jym66** [community/community#69587] [GitHub Education]
### Hi everyone,

I am applying for GitHub Global Campus, and the application has been approved. It shows "Congrats! Your request for @jym66 was approved and your benefits are now available", but I still cannot use this benefit.
![60d54571025ff7810a475e391d604606](https://github.com/community/community/assets/20449333/a085cd42-3474-4ceb-b850-9f0ca7092367)

**Comment by jym66** [comment]
It's been nine days and I haven't received any mail

**Comment by mantasu** [comment]
Hey, I had a similar issue, it is officially stated that the verification can take a few weeks, but if it's been 2-3 weeks you may just want to open a [ticket](https://support.github.com/request/education).

**Comment by jym66** [comment]
I opened a ticket but no one responded to me
